tests/foreman/ui/test_org.py

```diff
@@ -75,24 +104,20 @@ def test_positive_create_with_parent(self, parent_name):
         """
         org_name = gen_string('alpha')
         with Session(self.browser) as session:
-            make_org(session, org_name=parent_name)
-            make_org(
-                session,
-                org_name=org_name,
-                label=gen_string('alpha'),
-                desc=gen_string('alpha'),
-                parent_org=parent_name
-            )
-            self.assertIsNotNone(self.org.search(org_name))
+            for parent_name in generate_strings_list():
```

Comment:
I think we may have an issue here. For every iteration you will create a new 'parent' organization (line #109) but then you will create another new organization with `org_name=org_name` (which is fine, since this would be the first time it would be created) and associated with the new 'parent' organization. The second time you attempt to create a new organization with `org_name=org_name`, this should fail because we already have an organization named `org_name`. I think what we want to do here is perhaps create the 'parent' only once, outside the loop, and then create new organizations that are associated to the parent (assuming you can have many orgs associated to one parent?).